Sources/IO/Geometry/STLReader/index.js

```diff
@@ -77,6 +93,15 @@ function vtkSTLReader(publicAPI, model) {
     model.parseData = content;
 
     // Binary parsing
+    // Header
+    const headerData = new Uint8Array(content, 0, 80);
+    let headerStr = '';
+    for (let i = 0; i < 80; ++i) {
+      headerStr += String.fromCharCode(headerData[i]);
+    }
+    const header = parseHeader(headerStr);
```

Comment:
There is a better way to do it. See the XML readers.

Reply:
this?
https://github.com/Kitware/vtk-js/blob/master/Sources/IO/XML/XMLReader/index.js#L68-L95

Reply:
https://github.com/Kitware/vtk-js/blob/master/Sources/IO/Core/BinaryHelper/index.js#L7-L19

Reply:
done